diagnostic.md

```diff
@@ -5,74 +5,86 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+Without one it would not be possible for 2 (or more) people to access the same web app and
+interact with each other.
+It would also not be possible to save state and come back to the app later.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The controller users the view to to communicate with the model and fetch the data.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller in conjunction with the view.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Serializers are a class that we use instead of the view.  It’s choosing which part of the object and in what format to send back.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create READ UPDATE DESTROY
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+The model.
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+Get 1 --> Read
+Get index/all --> Read
+Post --> Create
+Patch --> Update
+Delete --> Destroy
+
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+1. Find the appropriate Route, which tells you the controller and method to use
+2. Go to the appropriate controller
+3. Within the controller, fire the method
+3. Method gets data from the model
+4. Model send the data back to the method
+5. Method returns the appropriate data in JSON or an error message.
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails-api
```

Comment:
You need a little more than this... `rails-api new app_name -T --database=postgresql`